source/hardware/controllers/native_instruments_traktor_kontrol_s2_mk3.rst

```diff
@@ -112,8 +112,7 @@ Mixxx mapping
 +------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------+
 | (25) LOW knob    | Low frequency filter                                                                                                        |                                                     |
 +------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------+
-| (26) Effect      | Quick effect superknob for the corresponding deck                                                                           |                                                     |
-| Superknob        |                                                                                                                             |                                                     |
+| 26. Effect knob  | Quick effect knob for the corresponding deck                                                                                |                                                     |
```

Comment:
Let's stick the current numbering format (here and below)
```suggestion
| (26) Effect knob  | Quick effect knob for the corresponding deck                                                                                |                                                     |
```

Reply:
Oh, the current version is already a mix of 12. and (13).
Do you mind unifying that? I'd prefer 12., or just a bold number **12**, maybe we don't need a . or () then.

Reply:
*ping*

Reply:
No I don't mind unifying, which one then? :)

Reply:
Either the numbers are in a separate column (too much hazzle if yo ask me), or 12. or **12**. I'd try both (`make html`) then choose what you prefer : )

Reply:
**12**. has my preference so let me fix that ;)